### DISCUSSION TITLE
逆天

### DISCUSSION THREAD

**Discussion by LyCecilion** [telecomadm1145/cmania#24] [General]
除了「逆天」之外还真不好说什么（）

希望这个项目大力发展下去！！！

**Comment by telecomadm1145** [comment]
感谢支持 目前项目可能暑假会接着做，现在项目还没怎么完成

**Reply by cowhorse05** [reply]
小白有点弄不懂怎么玩，或许是我cmake环境有问题，希望能在B站发个教程

**Reply by cowhorse05** [reply]
看到你在B站发的Linux版本教程了

**Reply by telecomadm1145** [reply]
> 小白有点弄不懂怎么玩，或许是我cmake环境有问题，希望能在B站发个教程

@Lgithubprogram 可以考虑使用预构建版本，依赖仅有libbass而已

**Reply by telecomadm1145** [reply]
另外希望有能力的还是尽量帮忙发现bug&更新功能/UI
